scripts/generate-rule-files/index.ts

```diff
@@ -159,6 +166,8 @@ export async function run(options: RunOptions = {}): Promise<void> {
     }
 
     logger.info(`Generating ${plugin.name} rules.`);
+    logger.logUpdate(logger.colors.yellow('  Loading plugin > Prettier'));
+    format('');
```

Comment:
```suggestion
```

Reply:
This is important, as it makes Prettier to run once. Just that line takes some seconds.